src/JoystickConfigState.cpp

```diff
@@ -101,12 +101,12 @@ void JoystickConfigState::update(float delta){
 
 	// joystick
 	if((input_manager->is_joystick_button_down(InputManager::R3, 0) &&
-		input_manager->is_joystick_button_down(InputManager::L3, 0))
+		input_manager->is_joystick_button_down(InputManager::L3, 0) && on_test)
```

Comment:
To keep the code organized, I guess it's better to group all inputs that happen `on_test` and all that happen `!on_test`